cmd/snap/cmd_quota.go

```diff
@@ -234,26 +344,44 @@ func (x *cmdQuota) Execute(args []string) (err error) {
 		fmt.Fprintf(w, "parent:\t%s\n", group.Parent)
 	}
 
-	fmt.Fprintf(w, "constraints:\n")
-
 	// Constraints should always be non-nil, since a quota group always needs to
-	// have a memory limit
+	// have atleast one limit set
 	if group.Constraints == nil {
 		return fmt.Errorf("internal error: constraints is missing from daemon response")
 	}
-	val := strings.TrimSpace(fmtSize(int64(group.Constraints.Memory)))
-	fmt.Fprintf(w, "  memory:\t%s\n", val)
 
-	fmt.Fprintf(w, "current:\n")
-	if group.Current == nil {
-		// current however may be missing if there is no memory usage
-		val = "0B"
-	} else {
-		// use the value from the response
-		val = strings.TrimSpace(fmtSize(int64(group.Current.Memory)))
+	fmt.Fprintf(w, "constraints:\n")
+
+	if group.Constraints.Memory != 0 {
+		val := strings.TrimSpace(fmtSize(int64(group.Constraints.Memory)))
```

Comment:
I see that you only moved this code, but (maybe for a follow-up PR) it would be nice to understand why `fmtSize` can return a string with extra spaces; if it does, it looks like a bug and we should fix it there, instead of calling `TrimSpace` here.

Reply:
Good question, I didn't think to much. I think you are right maybe we should look into why it's neccessary in a followup